Documentation/Installation/TutorialDdev.rst

```diff
@@ -136,13 +136,6 @@ Upon calling :bash:`ddev config` DDEV automatically created a database for
 you. DDEV also created a file called :file:`config/system/additional.php`
 in which it stored the database credentials for you.
 
-During the installation setup process TYPO3 created all the tables it needed.
-If you want to have a look at the database, you can run the following command:
-
-..  code-block:: bash
-
-    ddev launch -p
-
```

Comment:
ddev offers to install phpMyAdmin, might it be worth it to mention here?

Reply:
People use a variety of database browsers, which is why PhpMyAdmin was removed from DDEV core. If you prefer to mention a link here we can... https://ddev.readthedocs.io/en/stable/users/usage/database-management/#database-guis

Reply:
(DDEV does not offer to install PhpMyAdmin)

Reply:
yes, the link might be useful I think.

(With "offer" I meant that a hint how to install it is shown when the old command 'ddev launch -p` is entered, so it's no included functionality and shown only in narrow context. But you know it better than anyone else anyway :laughing: )

Reply:
I didn't even remember that `ddev launch -p` did that. It's great :)

Reply:
The new line with the link looks good for me!